### DISCUSSION TITLE
How to use Webpack caching when config file changes frequently

### DISCUSSION THREAD

**Discussion by gaojude** [webpack/webpack#16619] [General]
We all know that when the configuration file for webpack changes, the entire file system caching gets invalidated.

In our case, the configuration file needs to change very frequently, essentially each change in source code would imply a change in config file, so does this mean we can't benefit from file system caching at all?

Specifically, the changing config only affects one specific loader (babel-loader). Would be curious to know if we can configure it so that we can still cache & speed up other loaders prior to babel-loader (like ts-loader) using the file system caching?

**Comment by alexander-akait** [comment]
You can use `JSON.stringify` (stringify changes/path to files with changes) and use `cache.version`, so you will have cache for the each build, but yes, it will increase caches
